lib/mspec/runner/actions/constants_leak_checker.rb

```diff
@@ -18,6 +18,10 @@ class ConstantLeakError < StandardError
 end
 
 class ConstantsLeakCheckerAction
+  def initialize
+    @save = ENV['CHECK_LEAKS'] == 'save'
+  end
```

Comment:
I would move this check to upper level (to the `DottedFormatter` class) and pass `save` value as a parameter of `initialize`